docs/websites/cms/install-wordpress-ubuntu-18-04/index.md

```diff
@@ -44,48 +50,49 @@ location / {
 }
 {{< /file >}}
 
--   Make sure MySQL/MariaDB has a database set up for WordPress. If you do not have a WordPress database, create one:
+-    If you're using Apache, run the following commands to ensure that `mod_rewrite` is enabled:
 
-    1.  Log in to the MySQL command line as the root user. If use password authentication for the MySQL root user, use this command:
+        sudo a2enmod status
 
-            mysql -u root -p
+    If it isn't already enabled, enable a2enmod and restart Apache:
 
-        If you use `unix_socket` or `auth_socket` authentication for the MySQL root user (which is the default on Ubuntu 18.04), use this command instead:
+        sudo a2enmod rewrite
+        sudo systemctl restart apache2
+
+## Install WordPress
+
+### Prepare the WordPress Database
 
-            sudo mysql -u root
+WordPress stores blog posts and other content in your MySQL database, and you need to prepare the database before you can start using WordPress:
 
-    2.  Create the WordPress database:
+1.  Log in to the MySQL command line as the root user:
 
-        {{< highlight sql >}}
+        sudo mysql -u root
+
+2.  Create the WordPress database:
+
+    {{< highlight sql >}}
 CREATE DATABASE wordpress;
 {{< /highlight >}}
 
-    3.  Create a user and grant them privileges for the newly created `wordpress` database, replacing `wpuser` and `password` with the username and password you wish to use:
+3.  Create a user and grant them privileges for the newly created `wordpress` database, replacing `wpuser` and `password` with the username and password you wish to use:
 
-        {{< highlight sql >}}
+    {{< highlight sql >}}
 CREATE USER 'wpuser' IDENTIFIED BY 'password';
 GRANT ALL PRIVILEGES ON wordpress.* TO 'wpuser';
 {{< /highlight >}}
 
-    4.  Exit MySQL:
+4.  Exit MySQL:
 
-        {{< highlight sql >}}
+    {{< highlight sql >}}
 quit
 {{< /highlight >}}
 
-{{< note >}}
-This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you're not familiar with the `sudo` command, visit our [Users and Groups](/docs/tools-reference/linux-users-and-groups/) guide.
-
-All configuration files should be edited with elevated privileges. Remember to include `sudo` before running your text editor.
-
-Replace each instance of `example.com` in this guide with your site's domain name or IP.
-{{< /note >}}
-
-## Install WordPress
+### Download WordPress
 
 1.  Create a directory called `src` under your website's directory to store fresh copies of WordPress's source files. In this guide, the home directory `/var/www/html/example.com/` is used as an example. Navigate to that new directory:
 
-        sudo mkdir /var/www/html/example.com/src/
+        sudo mkdir -p /var/www/html/example.com/src/
```

Comment:
I'm not totally sure the `-p` is necessary, considering the "before you begin" steps include creating the parent directories as part of the stack, but I see what you're saying and I don't think it's a bad inclusion. just commenting here in case anyone comes looking for the why later